packages/ofType/src/ofType.ts

```diff
@@ -0,0 +1,70 @@
+
+import * as _ from 'lodash';
+
+import { IConstraintModule, ConstraintValue, ConstraintExecuter, IRuntimeContext, ValidationError, IValidator, ConstraintConfiguration } from '@configurable-validator/core'
+
+export class OfTypeModule implements IConstraintModule {
```

Comment:
this module comments are actually on the `format` module

Reply:
please explain

Reply:
I don't know what I meant